src/main/java/org/sasanlabs/service/vulnerability/xss/reflected/XSSWithHtmlTagInjection.java

```diff
@@ -85,4 +85,70 @@ public ResponseEntity<String> getVulnerablePayloadLevel3(
         }
         return new ResponseEntity<String>(payload.toString(), HttpStatus.OK);
     }
+
+    // Just adding User defined input(Untrusted Data) into div tag if doesn't contains
+    // eval(...) expression which evaluates the string expression and returns its value.
+    // Can be broken by various ways
+
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.REFLECTED_XSS,
+            description = "XSS_DIRECT_INPUT_DIV_TAG_AFTER_REMOVING_VALUES_CONTAINING_EVAL_EXPRESSION")
+    @VulnerableAppRequestMapping(value = LevelConstants.LEVEL_8, htmlTemplate = "LEVEL_1/XSS")
+    public ResponseEntity<String> getVulnerablePayloadLevel8(
+            @RequestParam Map<String, String> queryParams) {
+        String vulnerablePayloadWithPlaceHolder = "<div>%s<div>";
+        StringBuilder payload = new StringBuilder();
+        Pattern pattern = Pattern.compile("eval\\((.*?)\\)");
```

Comment:
can you please add the above black list to the pattern `[<]+[(script)(img)(a)]+.*[>]+` along with eval such that the level3 is vulnerable to eval but this level is not vulnerableto eval but to other payloads?
one more thing, please add the payload that can break this level.